packages/customize-widgets/src/components/customize-widgets/use-clear-selected-block.js

```diff
@@ -38,7 +38,8 @@ export default function useClearSelectedBlock( sidebarControl, popoverRef ) {
 					// 3. It should also not exist in the container, inspector, nor the popover.
 					! container.contains( element ) &&
 					! popoverRef.current.contains( element ) &&
-					! inspectorContainer.contains( element )
+					! inspectorContainer.contains( element ) &&
+					! element.closest( '[role="dialog"]' )
```

Comment:
This makes sense, but the purpose of `useClearSelectedBlock` confused me at first. 

I didn't realise we use this in addition to the normal block selection clearer.

I think it might be worth updating the description to mention some of the concrete needs for this hook. E,g, that it handles deselection when clicking areas outside the block list, like the preview area.

Reply:
Added in 2d47314.